Sources/PointFree/Admin/EpisodeCredit.swift

```diff
@@ -21,15 +21,10 @@ let showEpisodeCreditsMiddleware: Middleware<
   writeStatus(.ok)
     >=> respond({ _ in showEpisodeCreditsView })
 
-let redeemEpisodeCreditMiddleware: Middleware<
-  StatusLineOpen,
-  ResponseEnded,
-  Tuple3<User, User.Id?, Int?>,
-  Data
-  > =
-  filterMap(
-      over2(fetchUser(id:)) >>> sequence2 >>> map(require2),
-      or: redirect(to: .admin(.episodeCredits(.show)), headersMiddleware: flash(.error, "Could not find that user."))
+let redeemEpisodeCreditMiddleware
```

Comment:
is it not worth keeping the type here? does it not help at all?

Reply:
Makes it worse somehow :/